### PR TITLE
[5.0] remove Python 2 code

### PR DESCRIPTION
**Pull request by TaykYoku**
BEGINRELEASENOTES

CHANGE: remove Python 2 code

ENDRELEASENOTES

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,7 +3,6 @@ diraccfg
 autopep8==1.3.3
 coverage
 requests>=2.9.1
-six
```
Comment (fstagni) [inline requirements.txt:6]: is this file needed at all? I think we should align to use environment.yml like we do for DIRAC? Or none of the 2?

File: src/WebAppDIRAC/Core/HandlerMgr.py [hunk 2]
```diff
@@ -18,16 +10,13 @@
...
-
 from WebAppDIRAC.Lib import Conf
 from WebAppDIRAC.Lib.WebHandler import WebHandler, _WebHandler, WebSocketHandler
 from WebAppDIRAC.Core.CoreHandler import CoreHandler
 from WebAppDIRAC.Core.StaticHandler import StaticHandler
 
 
-@six.add_metaclass(DIRACSingleton)
-class HandlerMgr(object):
+class HandlerMgr:
```
Comment (fstagni) [inline src/WebAppDIRAC/Core/HandlerMgr.py]: You lost the metaclass. Why?

File: requirements.txt (hunk 1 above)
Comment (TaykYoku) [reply requirements.txt:6]: removed

File: src/WebAppDIRAC/Core/HandlerMgr.py (hunk 2 above)
Comment (TaykYoku) [reply src/WebAppDIRAC/Core/HandlerMgr.py]: well noticed, forgot to type it in python 3 style

**Review by fstagni — APPROVED** [review]
